schemas/ns10_cache_entry.fbs

```diff
@@ -0,0 +1,13 @@
+
+file_identifier "ns10";
+
+
+/// pylint: skip-file
+table CacheEntryFB {
+        time:double;
+	ttl:double;
+	expired:bool = false;
```

Comment:
Why do you want to be able to construct a flatbuffer of a cache entry which is already marked as expired?

Reply:
Here expired would imply that there should (possibly) be another value for this key - if not, use this old value instead and warn users. Apart from that, the value and time for the cache entry is anyways required for plotting historical graphs.

Although, the expired value is not needed explicitly, as this can always be calculated from the `time` and `ttl` (time to live) values. So, in principle, this data can be removed from the schema. Will have to check with NICOS people before doing that.

Reply:
Apparently,  the cache entries can also be expired manually, so we will need this field.

Reply:
That's a bit of a shame, I thought this might be a good use case for log compaction.

Reply:
Yes, we can still use log compaction. Log compaction also allows to delete the keys manually.